src/Model/ModelManager.php

```diff
@@ -629,6 +637,11 @@ protected function camelize($property)
         return str_replace(' ', '', ucwords(str_replace('_', ' ', $property)));
     }
 
+    protected function generatePrefix(): string
```

Comment:
Can you make this private

Reply:
I can't. I've extracted it into a separate method for testability, see https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/938/files#diff-c50f6d3b29e91b97ea76304d137c9929R1006

Reply:
👎 For adding an extension point just for tests. You could use a better assert to verify this.